server/controllers/patients/getAllPatientData.js

```diff
@@ -0,0 +1,37 @@
+const { boomify } = require('../../utils');
+
+const {
+  getHistoryLogs,
+  getPatientProfileData,
+} = require('../../database/queries');
+
+const getAllProfileData = async (req, res, next) => {
+  try {
+    const { patientId } = req.params;
```

Comment:
You need to validate `patientId` to check it's a valid number since if you try to get `/api/v1/patients/1a` this you will have an `Internal Server Error`

Reply:
done